apps/src/sites/studio/pages/blockly.js

```diff
@@ -1,4 +1,17 @@
-import Blockly from '@code-dot-org/blockly';
+import experiments from '@cdo/apps/util/experiments';
+
+import CDOBlockly from '@code-dot-org/blockly';
 import initializeCdoBlocklyWrapper from './cdoBlocklyWrapper';
 
-window.Blockly = initializeCdoBlocklyWrapper(Blockly);
+import Blockly from 'blockly/core';
```

Comment:
We should make sure the names reflect the source of Blockly (be it old, new, in-house, external, etc.), and also that it's clear that we use one or the other..